cli.js

```diff
@@ -201,12 +201,18 @@ function init(files) {
 function handlePaths(files) {
 	if (files.length === 0) {
 		files = [
+			'!node_modules',
 			'test.js',
 			'test-*.js',
 			'test/*.js'
 		];
 	}
 
+	// ignore all possible test files in node_modules folder
+	files = files.filter(function (file) {
+		return (file.indexOf('node_modules') === -1);
+	});
+
 	// convert pinkie-promise to Bluebird promise
 	files = Promise.resolve(globby(files));
```

Comment:
This line is where we trade the incoming glob strings for a Promise of all the files found by `globby` using said glob strings.

Your filter above should be moved down into the `then` method.